internal/graph/check.go

```diff
@@ -624,7 +625,7 @@ func (c *LocalChecker) buildCheckAssociatedObjects(req *ResolveCheckRequest, obj
 
 		typesys, ok := typesystem.TypesystemFromContext(ctx)
 		if !ok {
-			return nil, fmt.Errorf("typesystem missing in context")
+			return nil, fmt.Errorf("%w: typesystem missing in context", openfgaErrors.ErrUnknown)
```

Comment:
By the time you get here, you know very well the typesystem is in context, feel free to ignore the check until the change the structure for this.